docs/user/usage.rst

```diff
@@ -144,15 +144,37 @@ Simulate neuronal networks
 You can click on the :guilabel:`Simulate` button to start the simulation of your network.
 In the code editor you can have an insight into the generated script code (see below for further information).
 
-|
 
-.. image:: ../_static/img/screenshots/code-editor.png
+
+**Kernel settings**
+
+.. image:: ../_static/img/screenshots/kernel-settings.png
   :width: 360px
   :align: right
 
+The simulation parameters can be adjusted in the right sidebar.
+They are contained in the NEST Server code (more information below), so they will be passed to the NEST Server whenever a simulation is started.
+In the Kernel settings, the slider 'local number of threads' allows to set the number of processes used by the NEST Server.
+Please be aware that the shown number of threads does not match the number of processors used by the NEST Server machine.
+Therefore, selecting a number that is too large could lead to freezes on the NEST Server machine.
+
+It is possible to select the simulation resolution.
+Here, you should be aware of the created load on the NEST Server as well: small values for the resolution size create many calculations and data points.
+Therefore, selecting small values for the simulation resolution can lead to freezes and lags, so please be patient when you choose a small number. :)
+
+The seed of the random number generator can also be chosen.
+It is possible to choose a randomized seed.
+
+The simulation time can be set as well (in Milliseconds).
+
+
 
 **Code editor**
 
+.. image:: ../_static/img/screenshots/code-editor.png
+  :width: 360px
```

Comment:
Maybe more width (400px)

Reply:
The width of the image is already at 640px. I cropped the white space on the right (now 547px), maybe this helps a bit. If you think this is still an issue, please send me a message.